### PR TITLE
Feature/047 profile stats

### LINKED ISSUE #47: Stats in the profile

### PR DESCRIPTION
**Pull request by ExaCLe**
Closes #47

### CODE REVIEW

File: frontend/pages/playscreen.vue [hunk 1]
```diff
@@ -305,26 +316,26 @@ const checkAnswer = (direction) => {
...
+        Authorization: `Bearer ${token}`, // Füge Bearer Token zur Anfrage hinzu
       },
       body: payload,
     });
   } catch (error) {
-    // Show error toast
+    // Zeige Fehler-Toast
     toast.add({
       title: 'Error',
       description: 'Failed to submit test results. Please try again.',
```
Comment (aaronhampe) [inline frontend/pages/playscreen.vue]: I think we should provide error messages in german as well, do you agree?

File: frontend/pages/profile.vue [hunk 2]
```diff
@@ -30,6 +30,64 @@
...
+
+      <!-- Test Config Results Liste anzeigen, wenn das Laden abgeschlossen ist -->
+      <div v-else class="test-config-list">
+        <div
+          v-for="testConfig in testConfigResults"
+          :key="testConfig.id"
+          class="test-config-card"
+        >
+          <div class="card-header" @click="toggleCard(testConfig.id)">
+            <span class="card-title">Test ID: {{ testConfig.id }}</span>
```
Comment (aaronhampe) [inline frontend/pages/profile.vue]: Small Thing: Id call it Sehtest ID and I would also add the name of it so its easier to find the right test for the user

File: frontend/pages/playscreen.vue [hunk 3]
```diff
@@ -85,33 +85,35 @@
...
     <!-- Spielende-Anzeige -->
-    <div class="game-over" v-if="isGameOver && isTrisightMode">
-      <div class="final-score">Dein Score: {{ score }}</div>
-      <!-- Neue Buttons für Home und Neue Runde -->
-      <div class="button-group">
-        <NuxtLink to="/" class="button">Home</NuxtLink>
-        <button @click="startNewRound" class="button">Neue Runde</button>
-      </div>
-    </div>
+    <!-- Verwende das ResultsOverlay-Overlay -->
```
Comment (aaronhampe) [inline frontend/pages/playscreen.vue:88]: I dont know why but when I execute any test configs of my saved ones i dont get the ResultOverlay aftwerwards, the screen is just blank. Is that still in the working process or is there an error occuring? It works well for the Trisight Mode and i really like how it looks. Same with the profile view, very good job!!!

**Review by aaronhampe — COMMENTED** [review]
Just some small things and some questions i had. Great Job!

File: frontend/pages/playscreen.vue (hunk 3 above)
Comment (ExaCLe) [reply frontend/pages/playscreen.vue:88]: should be fixed

File: frontend/pages/playscreen.vue (hunk 1 above)
Comment (ExaCLe) [reply frontend/pages/playscreen.vue]: is translated

File: frontend/pages/profile.vue (hunk 2 above)
Comment (ExaCLe) [reply frontend/pages/profile.vue]: changed it

**Pull request merged**